core/state/database.go

```diff
@@ -87,6 +87,9 @@ type Trie interface {
 	// found in the database, a trie.MissingNodeError is returned.
 	TryDelete(key []byte) error
 
+	// DeleteACcount abstracts an account deletion from the trie.
+	DeleteAccount(key []byte) error
```

Comment:
Maybe `TryDeleteAccount` for alignment?

Reply:
Felix said that we should get rid of the `Try` in front of everything, so I figured that it would make sense to start the renaming now.

Reply:
Ah, gotcha